recipe/build.sh

```diff
@@ -4,34 +4,65 @@ CFLAGS=$(echo "${CFLAGS}" | sed "s/-mtune=[a-zA-Z0-9]*//g")
 # Avoid sorting LDFLAGS
 sed -i.bak 's/LDFLAGS := $(sort $(LDFLAGS))//g' common.mk
 
+
+# Multithreading
+MODEL="pthreads"
+
+
+# Map platform to BLIS target architecture 
+case $target_platform in
+    osx-64)
+        arch="intel64"
+        ;;
+    *-64)
+        arch="x86_64"
+        ;;
+    *-aarch64)
+        arch="arm64"
+        ;;
+    *-arm64)
+        arch="arm64"
+        ;;
+    #*-ppc64le)
+    #    arch="power9"
+    #   ;;
+    *)
+        echo "Unsupported architecture: $target_platform"
+        exit 1
+esac
+
+
+# Define target-specific options
 case $target_platform in
     osx-*)
-        export CC=$BUILD_PREFIX/bin/clang
-        ./configure --prefix=$PREFIX --enable-cblas --enable-threading=pthreads intel64
-        make CC_VENDOR=clang -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
+        export CC_VENDOR=clang
+        EXTRA=""
         ;;
     linux-*)
         ln -s `which $CC` $BUILD_PREFIX/bin/gcc
-        export CC=$BUILD_PREFIX/bin/gcc
-        ./configure --prefix=$PREFIX --enable-cblas --enable-threading=pthreads x86_64
-        make CC_VENDOR=gcc -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
+        export CC_VENDOR=gcc
+        EXTRA=""
         ;;
     win-*)
-        export LIBPTHREAD=
-        ./configure --disable-shared --enable-static --prefix=$PREFIX --enable-cblas --enable-threading=pthreads --enable-arg-max-hack x86_64
-        make -j${CPU_COUNT}
-        make install
-        make check -j${CPU_COUNT}
- 
-        ./configure --enable-shared --disable-static --prefix=$PREFIX --enable-cblas --enable-threading=pthreads --enable-arg-max-hack x86_64
-        make -j${CPU_COUNT}
-        make install
-        mv $PREFIX/lib/libblis.lib $PREFIX/lib/blis.lib
-        mv $PREFIX/lib/libblis.a $PREFIX/lib/libblis.lib
-        mv $PREFIX/lib/libblis.*.dll $PREFIX/bin/
+        export LIBPTHREAD=""
+        EXTRA="--enable-arg-max-hack --disable-shared --enable-static"
         ;;
 esac
+
+
+# General case
+./configure --prefix=$PREFIX --enable-cblas --enable-threading="$MODEL" $EXTRA $arch
+make -j${CPU_COUNT}
+make install
+make check -j${CPU_COUNT}
+
+
+# Windows-specific shenanigans (builds twice; first static above, then shared below)
+case $target_platform in win-*)
```

Comment:
Is this possible ?
I thought that `bld.bat` would be used for Windows, not `build.sh`.